### PR TITLE
delta-1284 add http headers filter

### PR DESCRIPTION
**Pull request by MihailTymoshenko**
Replace this text with a description of your proposed change.

[JIRA Ticket](https://bt.smartling.net/browse/DELTA-1284)

## Squash Commit Guidelines

This is a reminder on how to format your sqaush commit message. Delete from your PR after noting below.

### Message Header

The message header is a single line that contains succinct description of the change containing a type: subject. For example:

`feat: add HttpHeadersFilter`

### CODE REVIEW

File: smartling-api-commons/src/main/java/com/smartling/api/v2/client/HttpHeadersFilter.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        this.httpHeaders = new HashMap<>();
+        this.httpHeaders.put(key, value);
+    }
+
+    public HttpHeadersFilter(String key1, String value1, String key2, String value2)
+    {
+        this.httpHeaders = new HashMap<>();
+        this.httpHeaders.put(key1, value1);
+        this.httpHeaders.put(key2, value2);
+    }
```
Comment (foo4u) [inline smartling-api-commons/src/main/java/com/smartling/api/v2/client/HttpHeadersFilter.java:29]: I don't think this constructor adds much value.

**Review by foo4u — CHANGES_REQUESTED** [review]
Please add a test for this new filter.

**Comment by MihailTymoshenko** [comment]
> Please add a test for this new filter.
test added

**Review by foo4u — APPROVED** [review]
LGTM

**Review by vbarhatov — COMMENTED** [review]
nice

**Pull request merged**